CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
```suggestion
# Changelog

All notable changes to Alacritty are documented in this file.
The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` and `Removed`.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
```

Reply:
I'd assume that's crossfont in that case. Also, the header were from copypasta, so we should fix it as well?

Reply:
Yeah, sorry. I'm fine with copying it from copypasta too, yes. We probably just changed it in Alacritty after adding it to copypasta.